SharedLibs/TileMapHelper/karma.conf.js

```diff
@@ -0,0 +1,42 @@
+module.exports = function (config) {
```

Comment:
do we need a whole different karma conf for the TileMapHelper?

Reply:
Given that the other Karma run with the bundled lib, I guess it's necessary. This configuration has the advantage of allowing to write tests in TypeScript.